packages/nodejs/READMEV2.md

```diff
@@ -389,4 +389,77 @@ const blockHash = 'fe88ff35454079c3df11d8ae13d5777babd61f28be58494efe51b6593e307
 
 const state: HexString = await clientV2.instanceStateLookup(blockHash);
 ...
+```
+
+### getIdentityProviders
+Get the identity providers registered as of the end of a given block as a stream
+
+If a blockhash is not supplied it will pick the latest finalized block. An optional abortsignal can also be provided that closes the stream.
+
+```js
+const blockHash = "7f7409679e53875567e2ae812c9fcefe90ced8961d08554756f42bf268a42749";
+const ips = await client.getIdentityProviders(blockHash);
+
+for await (const ip of ips) {
+    console.log(ip.ipDescription);
+}
+```
+
+## getAnonymityRevokers
+Get the anonymity revokers registered as of the end of a given block as a stream.
+
+If a blockhash is not supplied it will pick the latest finalized block. An optional abortsignal can also be provided that closes the stream.
+```js
+const blockHash = "7f7409679e53875567e2ae812c9fcefe90ced8961d08554756f42bf268a42749";
+const ars = await client.getAnonymityRevokers(blockHash);
+
+for await (const ar of ars) {
+    console.log(ar.ipDescription);
+}
+...
+```
+
+## getBlocksAtHeight
+Get a list of live blocks at a given height.
+
+If a blockhash is not supplied it will pick the latest finalized block. An optional abortsignal can also be provided that closes the stream.
+
+It can accept an absolute height:
+```js
+const blocks = await client.getBlocksAtHeight(100n);
+...
+```
+Or it can accept a relative height:
+```js
+const request: BlocksAtHeightRequest = {
+    // Genesis index to start from.
+    genesisIndex: 1;
+    // Height starting from the genesis block at the genesis index.
+    height: 100n;
+    // Whether to return results only from the specified genesis index (`true`),
+    // or allow results from more recent genesis indices as well (`false`).
+    restrict: true;
+}
+const blocks = await client.getBlocksAtHeight(request);
+```
+
+## getBlockInfo
+Retrieves information about a specific block.
+```js
+const blockHash = "7f7409679e53875567e2ae812c9fcefe90ced8761d08554756f42bf268a42749";
+const blockInfo: BlockInfo = await client.getBlockInfo(blockHash);
+const transactionsCount = blockInfo.transactionCount;
+...
+```
+
+## getBakerList
+Retrieves a stream of ID's for registered bakers on the network at a specific block.
```

Comment:
```suggestion
Retrieves a stream of ID's for registered bakers on the network at a specific block.

If a blockhash is not supplied it will pick the latest finalized block. An optional abort signal can also be provided that closes the stream.
```